CHAD-4316 Remove "alarmState" custom attribute from zwave smoke alarm

greens opened the pull request:
We're moving away from custom attributes. This PR should match the work done to remove this attribute from hubcore.

greens committed:
CHAD-4316 Remove "alarmState" custom attribute from zwave smoke alarm

We're moving away from custom attributes. This PR should match the work done to remove this attribute from hubcore.

tpmanley commented on devicetypes/smartthings/zwave-smoke-alarm.src/zwave-smoke-alarm.groovy:
```diff
@@ -37,13 +35,17 @@ metadata {
 				attributeState("tested", label:"TEST", icon:"st.alarm.smoke.test", backgroundColor:"#e86d13")
 			}
 		}
+		standardTile("co", "device.carbonMonoxide", width:6, height:4, inactiveLabel: false, decoration: "flat") {
```
Need to update `details`

greens committed:
CHAD-4316 Remove "alarmState" custom attribute from zwave smoke alarm

We're moving away from custom attributes. This PR should match the work done to remove this attribute from hubcore.
